build.gradle

```diff
@@ -109,12 +110,48 @@ task sourcesJar(type: Jar) {
     from sourceSets.main.allJava
 }
 
+task javadocJar(type: Jar) {
+   classifier = 'javadoc'
+   from javadoc.destinationDir
+}
+
 publishing {
     publications {
         shadow(MavenPublication) { publication ->
             project.shadow.component(publication)
             artifact sourcesJar
         }
+        sonatype(MavenPublication) { publication ->
+            artifactId = "opendistro_sql_jdbc"
```

Comment:
'opendistro-sql-jdbc' is already in settings.gradle. is this still needed?

Reply:
also please confirm this publishes the shadowJar artifact i.e. the fat jar with dependencies shaded in.